packages/react-ui-core/src/Ratings/__tests__/RatingBar-test.js

```diff
@@ -1,72 +1,29 @@
 import React from 'react'
-import { mount } from 'enzyme'
-import theme from './mocks/theme'
+import { shallow } from 'enzyme'
 import ThemedRatingBar from '../RatingBar'
 
 const RatingBar = ThemedRatingBar.WrappedComponent
 
 describe('RatingBar', () => {
-  it('has a default maxScore', () => {
-    const wrapper = mount(<RatingBar uniqueId="foo" />)
-    expect(wrapper.find('svg').length).toBe(5)
-  })
-
   it('renders empty Ratings when rating is not provided', () => {
-    const wrapper = mount(<RatingBar uniqueId="foo" maxScore={5} />)
-    expect(wrapper.find('svg').length).toBe(5)
-  })
-
-  describe('classNames', () => {
-    it('renders a "blue" className when color blue is passed', () => {
-      const wrapper = mount(<RatingBar
-        uniqueId="foo"
-        maxScore={1}
-        theme={theme}
-        fillColor="blue"
-      />)
-      expect(wrapper.find('.RatingBar_Item-blue').length).toBe(2)
-    })
-
-    it('does not render a "red" className when color does not exist in theme', () => {
-      const wrapper = mount(<RatingBar
-        uniqueId="foo"
-        maxScore={1}
-        theme={theme}
-        fillColor="red"
-      />)
-      expect(wrapper.find('.RatingBar_Item-red').length).toBe(0)
-    })
+    const wrapper = shallow(<RatingBar maxScore={5} />)
+    expect(wrapper.html()).toMatchSnapshot()
```

Comment:
If only testing the rendered HTML of the component, consider using enzyme's `render` instead of `shallow` (but I'm actually not sure what the prefered method is).